server/opts.go

```diff
@@ -986,6 +987,8 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 		o.PortsFileDir = v.(string)
 	case "prof_port":
 		o.ProfPort = int(v.(int64))
+	case "prof_tags":
```

Comment:
I don't think this is needed, if you turn on profiling we will add in the tags.

Reply:
I can see an argument for both. If someone was sharing with us let's say would want all the information so that we could tie together any NATS cli triage, e.g. `nats stream info` with any profiling they may share.